File: Python3/Rating Problems.py
#https://open.kattis.com/problems/ratingproblems

n = input().split(" ")
sum = 0
for i in range(int(n[1])):
    sum += int(input())
print("{0} {1}".format(((int(n[0]) - int(n[1])) * -3.0 + sum)/ int(n[0]),((int(n[0]) - int(n[1])) * 3.0 + sum)/ int(n[0])))

File: C/Rating Problems.c
//https://open.kattis.com/problems/ratingproblems
#include <stdio.h>
//#include <string.h>
//#include <stdlib.h>

int main() {
	int n;
	scanf("%d", &n);
	int k;
	scanf("%d", &k);
	int sum = 0;
	
	for (int i = 0; i < k; i++)
	{
		int c;
		scanf("%d", &c);
		sum += c;
	}
	
	printf("%.4f %.4f", (sum+(n-k)*(-3.0))/n, (sum+(n-k)*(3.0))/n);
  return 0;
}
